ui/raidboss/emulator/data/network_log_converter/LineEvent0x01.ts

```diff
@@ -1,16 +1,24 @@
 import LineEvent from './LineEvent';
 import EmulatorCommon from '../../EmulatorCommon';
+import LogRepository from './LogRepository';
 
 // Zone change event
 export class LineEvent0x01 extends LineEvent {
-  constructor(repo, line, parts) {
-    super(repo, line, parts);
-    this.zoneId = parts[2].toUpperCase();
-    this.zoneName = parts[3];
-    this.zoneNameProperCase = EmulatorCommon.properCase(this.zoneName);
+  public properCaseConvertedLine = '';
+
+  public get zoneID(): string {
```

Comment:
```suggestion
  public get zoneId(): string {
```

To match other uses of `zoneId` and `thingId` elsewhere.